src/extras-openmoji/climate-environment/E2C3.svg

```diff
@@ -0,0 +1,52 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```

Comment:
too big, also probably recognisable if you're familiar but I thought it was an oil rig or similar